README.md

```diff
@@ -45,42 +37,11 @@ const sandboxProfileClient = new SandboxProfileClientBuilder()
   .withPemString(PEM)
   .build();
 
-const ageVerification = new SandboxAgeVerificationBuilder()
-  .withDateOfBirthString('1980-01-01')
-  .withAgeOver(18)
-  .build();
+const yotiClient = new Client(SANDBOX_CLIENT_SDK_ID, PEM);
+```
 
-const tokenRequest = new TokenRequestBuilder()
-  .withRememberMeId('some remember me ID')
-  .withGivenNames('some given names')
-  .withFamilyName('some family name')
-  .withFullName('some full name')
-  .withDateOfBirthString('1980-01-01')
-  .withAgeVerification(ageVerification)
-  .withGender('some gender')
-  .withPhoneNumber('some phone number')
-  .withNationality('some nationality')
-  .withStructuredPostalAddress(JSON.stringify({
-    building_number: 1,
-    address_line1: 'some address',
-  }))
-  .withBase64Selfie('some base64 encoded selfie')
-  .withEmailAddress('some@email')
-  .withDocumentDetails('PASSPORT USA 1234abc')
-  .build();
+> Note: To point the client at the sandbox, set environment variable `YOTI_CONNECT_API` to https://api.yoti.com/sandbox/v1
```

Comment:
Do you think we should make this more prominent somehow? (more a question for @jlmartyres)

Also not sure if it's worth specifying Yoti Client here, since we have Sandbox Client in the same block.
```suggestion
> Note: To point the Yoti client at the sandbox, set environment variable `YOTI_CONNECT_API` to https://api.yoti.com/sandbox/v1
```

Reply:
I've moved this to the configuration bullet points in c1cf35c